apriltags_ros/src/apriltag_detector.cpp

```diff
@@ -99,6 +106,17 @@ AprilTagDetector::AprilTagDetector(ros::NodeHandle& nh, ros::NodeHandle& pnh) :
   pnh.param<float>("plane_angle_threshold", plane_angle_threshold_,0.0872665f);
   plane_angle_threshold_ = std::max(0.0f, std::min(90.0f, plane_angle_threshold_));
 
+
+  pnh.param<float>("tf_pose_acceptance_error_range in radians", tf_pose_acceptance_error_range_, 4);
```

Comment:
Is this a typo? Looks like underscore are missing

Reply:
If it is in radians, the default should not be 4.

Reply:
yes fixed typos